Userland/Applications/DisplaySettings/MonitorSettingsWidget.h

```diff
@@ -43,7 +43,7 @@ class MonitorSettingsWidget final : public GUI::SettingsWindow::Tab {
     ErrorOr<void> create_resolution_list();
     ErrorOr<void> load_current_settings();
     ErrorOr<void> generate_resolution_strings();
-    ErrorOr<void> selected_screen_index_or_resolution_changed();
+    ErrorOr<void> selected_screen_index_or_resolution_changed(bool screen_index_changed);
```

Comment:
Can you use an enum for this? https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#right-8

Reply:
Would it be better to use an enum or to refactor into "selected_screen_index_changed" and "selected_screen_resolution_changed", wherein one would call the other? I'm struggling to figure out what an enum for this should be called, given it just prepends a very small piece of code to the front of the function tbh

Reply:
1enum class DidScreenIndexChange { No, Yes };` is probably not the best name, but it's good enough :) Let's just do that for now, and then do larger changes in a later PR, if you want to change more.

Reply:
Pushed the changes. Is the new code alright? Wasn't sure if the enum was supposed to go outside the class definition or inside.

Reply:
Either is fine, it doesn't matter too much. (Personally, I'd put it in the class right above the method it belongs to, but it's fine as-is too.)

Can you squash the last commit into the previous commit though, so that there isn't an intermediate state with the bool? (`git rebase -i master`, put a `f` before the last commit, save, then `git push --force-with-lease`)

Reply:
Commit squashed and all checks passed. Had to do an empty amend to force the CI to repeat itself, there were some apt problems in two of the ubuntu x86_64 builds.